www/src/components/Settings.tsx

```diff
@@ -28,6 +28,7 @@ const Settings = () => {
       position="right"
       show={showSettings}
       variant="light"
+      onClose={closeSettings}
```

Comment:
[clarification] Little usability improvement to ensure the "Settings" `Sheet` closes on Escape key press on the Paragon documentation website.